tests/test_endpoints.py

```diff
@@ -0,0 +1,232 @@
+from datetime import datetime
+
+import pytest
+
+from tests.conftest import TEST_PAGE_NAME
+
+
+@pytest.fixture(scope="module")
+def session_page_id(vcr) -> str:
+    with vcr.use_cassette("test_pages_create.yaml") as cass:
+        response = cass._serializer.deserialize(cass.data[0][1]["content"])
+        return response["id"]
```

Comment:
What is the difference between this fixture and `conftest.page_id`?

Reply:
I added a couple of comments and moved the fixtures in conftest for clarity. 
In short, for creating the session_page_id, which is the page where the tests will run, we need a parent (see test_pages_create), so it's necessary to use both of them. The only function of page_id is to be a parent in the API call.
Feedbacks are welcomed for a clearer name :)

Reply:
Makes sense! Maybe we could rename the environment variable and fixture from  `NOTION_TEST_PAGE_ID` to `PARENT_PAGE_ID`? We could even drop the `session_` prefix if we go that way, `parent_page_id` and `page_id` seem clear enough to me.